features/kinderen.feature

```diff
@@ -10,29 +10,39 @@ Functionaliteit: Kinderen van een persoon raadplegen
 
     @gba
     Scenario: de naam van een kind is gecorrigeerd
-      Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-      | Categorie | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Onjuist (84.10) |
-      | 9         | William           |                     | Postma                |                 |
-      | 59        | William           | de                  | Vries                 |                 |
-      | 59        | William           | de                  | Boer                  | O               |
-      | 9         | Sebastiaan        | de                  | Boer                  |                 |
-      | 9         | Walter            | de                  | Boer                  |                 |
-      | 59        | Walter            |                     | Messeritz             | O               |
+      Gegeven de persoon met burgerservicenummer '999996150' heeft een 'kind' met de volgende gegevens
```

Comment:
dit is een persoon die al in de proefomgeving (en dus de locale database) zit. Zorg je er in de automation code voor dat de persoon niet dubbel wordt gecreëerd? Of zouden we hier het burgerservicenummer moeten wijzigen naar een test-bsn die nog niet in de database staat?

Reply:
voordeel van een nieuwe persoon maken is in dit geval dat we in gegeven en dan stappen het tweede en derde kind weg kunnen laten. die zijn niet relevant voor het scenario en alleen toegevoegd om te zorgen dat het correct werkt

Reply:
> dit is een persoon die al in de proefomgeving (en dus de locale database) zit. Zorg je er in de automation code voor dat de persoon niet dubbel wordt gecreëerd? Of zouden we hier het burgerservicenummer moeten wijzigen naar een test-bsn die nog niet in de database staat?

De tweede optie. De automation code zou de check op bsn kunnen doen, maar dan komt de log niet meer overeen met de scenario

Reply:
> voordeel van een nieuwe persoon maken is in dit geval dat we in gegeven en dan stappen het tweede en derde kind weg kunnen laten. die zijn niet relevant voor het scenario en alleen toegevoegd om te zorgen dat het correct werkt

Eens, Volgens mij zouden we deze manier van werken hanteren. We willen toch niet meer afhankelijk zijn van bestaande testgevallen ? Bijkomend voordeel is inderdaad dat je testgevallen kan beperken tot wat relevant is voor het scenario.